src/test/java/seedu/address/model/person/NameTest.java

```diff
@@ -37,4 +36,18 @@ public void isValidName() {
         assertTrue(Name.isValidName("Capital Tan")); // with capital letters
         assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
     }
+
+    @Test
+    public void capitalise() {
+        // null name
+        assertThrows(NullPointerException.class, () -> Name.isValidName(null));
+
+        assertEquals(Name.capitalise("a b c d e"), "A B C D E");
```

Comment:
Perhaps you can consider adding one more test case to ensure the rest of the word is in lowercase? e.g. ``CLIffOrd KAlEB toh jiNg zHe``` should give ```Clifford Kaleb Toh Jing Zhe```?

Reply:
Roger that